apps/src/templates/MultipleSectionsAssigner.jsx

```diff
@@ -154,6 +155,30 @@ const MultipleSectionsAssigner = ({
     );
   };
 
+  const assignToSectionWithConfirmation = (
+    sectionId,
+    courseId,
+    courseOfferingId,
+    courseVersionId,
+    scriptId
+  ) => {
+    onAssignSuccess
+      ? assignToSection(
+          sectionId,
+          courseId,
+          courseOfferingId,
+          courseVersionId,
+          scriptId
+        ).then(onAssignSuccess)
```

Comment:
I never thought of assignToSection as a promise, but this makes sense! Cool beans 😎

Reply:
I didn't either but I believe that's the way it ends up being handled!